src/backends/llvm-backend.ts

```
@@ -95,122 +120,291 @@
+        gen.emit(`  ret i1 true`);
+    }
+  }
+
+  private compileTransformToLLVM(transform: SafeTransform, gen: LLVMIRGenerator, valueReg: string): void {
+    switch (transform.type) {
+      case 'identity':
+        gen.emit(`  ret double ${valueReg}`);
+        break;
+        
+      case 'constant':
+        const constValue = typeof transform.value === 'number' ? transform.value : 0;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)